doc/source/user/configoptions.rst

```diff
@@ -555,6 +572,73 @@ Networking options
 
       Filename of a text file with "key: value" HTTP headers.
 
+-  .. config:: GDAL_HTTP_CONNECTTIMEOUT
+      :choices: <seconds>
+      :since: 2.2
+
+      Maximum delay for connection to be established before being aborted.
+
+-  .. config:: GDAL_HTTP_COOKIE
+      :since: 2.0
+
+      Cookie(s) to send. See https://curl.se/libcurl/c/CURLOPT_COOKIE.html
+
+-  .. config:: GDAL_HTTP_COOKIEFILE
+      :since: 2.4.0
+
+      File name to read cookies from. See https://curl.se/libcurl/c/CURLOPT_COOKIEFILE.html
+
+-  .. config:: GDAL_HTTP_COOKIEJAR
+      :since: 2.4.0
+
+      File to which cookies should be written. See https://curl.se/libcurl/c/CURLOPT_COOKIEJAR.html
+
+-  .. config:: GDAL_HTTP_NETRC
+      :choices: YES, NO
+      :default: YES
+      :since: 1.11.0
+
+      Controls if an available ``.netrc`` file is used.
+
+-  .. config:: GDAL_HTTP_NETRC_FILE
+      :choices: <filename>
+      :since: 3.7.0
+
+      Sets the location of a ``.netrc`` file.
+
+-  .. config:: GDAL_HTTP_ENABLE_ADVISE_READ
+      :choices: YES, NO
+      :default: YES
+      :since: 3.7.0
+
```

Comment:
I would remove that. That's to disable an optimization. Very advanced

Reply:
Agreed, my mistake. Removed.